app/routes/followers.rb

```diff
@@ -1,36 +1,25 @@
 helpers do
   def follower_query
-    with_organization_and_course email: params[:email]
-  end
-
-  def follower_query_from_body
-    with_organization_and_course email: json_body[:email]
+    with_organization_and_course email: current_user.uid
```

Comment:
`uid` is not always and email, so either you are passing a wrong parameter or you are not naming this `email`key properly.